src/platforms/web/runtime/modules/dom-props.js

```diff
@@ -51,7 +53,7 @@ function updateDOMProps (oldVnode: VNodeWithData, vnode: VNodeWithData) {
     } else if (key === 'innerHTML' && isSVG(elm.tagName) && isUndef(elm.innerHTML)) {
       // IE doesn't support innerHTML for SVG elements
       svgContainer = svgContainer || document.createElement('div')
-      svgContainer.innerHTML = `<svg>${cur}</svg>`
+      svgContainer.innerHTML = maybeCreateDangerousSvgHTML(cur)
```

Comment:
This file actually handles the setting of normal `innerHTML` as well (down at line 72) - it's just we didn't have a dedicated branch to it.